### DISCUSSION TITLE
accelerate-and-secure-with-cloudflared/

### DISCUSSION THREAD

**Discussion by giscus** [n0vad3v/blog-comments#1] [Ideas]
# accelerate-and-secure-with-cloudflared/

This article is also available in English, at Use Cloudflare Argo Tunnel (cloudflared) to accelerate and protect your website. 本文于 2024-03-01 更新，更新了新版本的 cloudflared 部署和安装方式 是的，这是一篇记录 cloudflared 的使用历程和一些思路的文章，关于 Cloudflare 的 Argo

https://nova.moe/accelerate-and-secure-with-cloudflared/

**Comment by 14790897** [comment]
不错

**Comment by senshinya** [comment]
最后一段很有感触

**Comment by tangbojin** [comment]
我嘿嘿嘿白嫖得正香，最后一段骂的我头都抬不起来

**Comment by henovo13** [comment]
专门连接上来为最后一段鼓掌的，任何一个充分理解了这段话的含义的中国人，必定早已是遍体鳞伤。
愚民教育在大环境下已经是默认的必然，现代的人类并不比远古的人类头脑更发达，而是因为见识，知识是通过见识进行积累，这是人类进步的必然步骤。
继续封锁，重新进入上世纪那个不堪回首的年代，才能新生，咱们的子孙后代才不会像我们这样，成为现代文明的残废。

**Reply by 14790897** [reply]
.。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。

**Comment by Pilo-pillow** [comment]
看到最后一段话真的很难过，回想到对计算机兴趣达到顶峰的小时候，由于信息缺乏和没有相应的氛围导致自己的发展真的很慢，现在也是这样，国外行业里里外外都在使用和讨论AI相关内容，国内基本只有业内人员会去使用
